ui/Rhino/AGS/dev/AGS_check_loadpath_cmd.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import print_function
+from __future__ import absolute_import
+from __future__ import division
+
+import scriptcontext as sc
+
+import compas_rhino
+
+
+__commandname__ = "AGS_check_loadpath"
+
+
+def RunCommand(is_interactive):
+
+    if 'AGS' not in sc.sticky:
+        compas_rhino.display_message('AGS has not been initialised yet.')
+        return
+
+    proxy = sc.sticky['AGS']['proxy']
+    scene = sc.sticky['AGS']['scene']
+    form = scene.find_by_name('Form')[0]
+    force = scene.find_by_name('Force')[0]
+
+    proxy.package = 'compas_ags.ags.loadpath'
+
+    lp = proxy.compute_loadpath_proxy(form.diagram.data, force.diagram.data)
```

Comment:
with the recent updates to compas it should not be necessary anymore to use a `_proxy` wrapper for this. it should be possible to use `compute_loadpath` directly (through the proxy server).

currently the only functions that require a `_proxy` wrapper are those that don't return anything but rather update the input data structures...

Reply:
woo... it works... 
(don't fully get why... but it works... :-D )